lib/vagrant-openshift/action.rb

```diff
@@ -153,6 +153,18 @@ def self.repo_sync_sti(options)
         end
       end
 
+      def self.repo_sync_jenkins(options)
```

Comment:
In August we furthermore made the choice not to invest time in making `vagrant-openshift` better but instead in focusing development time on the `origin-ci-tool`. We are set to see `oct` make it's debut in supporting Origin PR tests this sprint. The tool provides `oct sync` which aims to provide the generic functionality we need.